Source/WebKit/WebProcess/Plugins/PDF/PDFPlugin.mm

```diff
@@ -1367,6 +1367,8 @@ static bool getEventTypeFromWebEvent(const WebEvent& event, NSEventType& eventTy
     // FIXME: How can always returning true without searching if passed a max of 0 be right?
     // Even if it is right, why not put that special case inside countFindMatches instead of here?
     bool foundMatch = !maxMatchCount || countFindMatches(target, options, maxMatchCount);
+    if (options.contains(WebCore::DoNotSetSelection))
```

Comment:
Not in this PR, but mind making `FindOptionFlag` an `enum class FindOptionFlag : uint16_t` at some point?